### DISCUSSION TITLE
Uploading audio clips to article

### DISCUSSION THREAD

**Discussion by yheuhtozr** [forem/forem#19152] [Feature Requests]
There was a request from my users to directly upload audio files to the article, not only images, so that you can click and play the (usually small piece of) clip in the middle of article with a compact UI that, for example, looks as below.

![audio UI](https://user-images.githubusercontent.com/84892012/219872692-0287bb40-5814-431d-9c28-3410abec209d.png)

I'd appreciate if it'd be available vanilla without an extra detour uploading them to an embeddable external services.

**Comment by rc2168st** [comment]
we need this

**Comment by epikijetesantakalu** [comment]
実装されないかな~

**Comment by tkt2020-scratch** [comment]
We need this
